addons/rating_project_issue/models/project_issue.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class ProjectIssue(models.Model):
+
+    _name = "project.issue"
+    _inherit = ['project.issue', 'rating.mixin']
+
+    @api.multi
+    def write(self, values):
+        if 'stage_id' in values and values.get('stage_id', False):
+            template = self.env['project.task.type'].browse(values.get('stage_id')).rating_template_id
+            if template:
+                rated_partner_id = self.user_id.partner_id
+                partner_id = self.partner_id
+                if partner_id and rated_partner_id:
+                    self.rating_send_request(template, partner_id, rated_partner_id)
+        return super(ProjectIssue, self).write(values)
+
+
+class Project(models.Model):
+
+    _inherit = "project.project"
+
+    @api.multi
+    @api.depends('percentage_satisfaction_task', 'percentage_satisfaction_issue')
+    def _compute_percentage_satisfaction_project(self):
+        super(Project, self)._compute_percentage_satisfaction_project()
+        Rating = self.env['rating.rating']
+        for record in self:
+            if record.use_tasks or record.use_issues:
+                # built the domain according the project parameters (use tasks and/or issues)
+                res_models = []
+                domain = []
+                if record.use_tasks:
+                    res_models.append('project.task')
+                    domain += ['&', ('res_model', '=', 'project.task'), ('res_id', 'in', record.tasks.ids)]
+                if record.use_issues:
+                    res_models.append('project.issue')
+                    domain += ['&', ('res_model', '=', 'project.issue'), ('res_id', 'in', record.issue_ids.ids)]
+                if len(res_models) == 2:
+                    domain = ['|'] + domain
+                domain = [('rating', '>=', 0)] + domain
```

Comment:
Don't forget to add the '&' in front. It is better to format explicit domains than use the implicit '&'; easier to manipulate or inherit.